client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/AdditionalInfo.java

```diff
@@ -65,6 +65,21 @@ public StatementDescription(String line, String method) {
      */
     private final Map<String, Deque<StatementDescription>> lastExecutedStatementStacks = new ConcurrentHashMap<>();
 
+    /**
+     * To keep track of external service hosts called by the SUT.
+     * When captured mock hostname will be empty, till the WireMock instance
+     * initiated.
+     */
+    private final Set<ExternalServiceInfo> externalServices = new CopyOnWriteArraySet<>();
+
+    public void addExternalService(ExternalServiceInfo hostInfo) {
```

Comment:
move these public setter/getter after the private fields (ie, be consistent with all the other getters/setters location)